src/views/DetailedView/BookingSystem/index.tsx

```diff
@@ -13,39 +13,53 @@ import {
   GreyDivider,
   PriceWrapper,
 } from "./bookingSystem.styled";
-import { FunctionComponent } from "react";
+import { FunctionComponent, useState } from "react";
 import { useExchangeRateContext } from "../../../shared/ExchangeRateContext/useExchangeRateContext.tsx";
 import { getPriceInPln } from "../../../shared/utils/getPriceInPln.ts";
+import { useCalendarDates } from "./useCalendarDates.tsx";
+import { getNumberOfDays } from "./getNumberOfDays.tsx";
 
 interface Props {
   priceInEur: string | number;
 }
 export const BookingSystem: FunctionComponent<Props> = ({ priceInEur }) => {
   const { exchangeRate } = useExchangeRateContext();
-
+  const { value, handleChange } = useCalendarDates();
+  const [isRange, setIsRange] = useState(true);
   if (!exchangeRate) {
     return null;
   }
-  const priceInPln = getPriceInPln(priceInEur, exchangeRate.plnPerEur);
+
+  const handleCheckbox = () => {
+    setIsRange(!isRange);
+  };
+
+  const priceInPln = getPriceInPln(priceInEur, exchangeRate.plnPerEur);  const numberOfDays = getNumberOfDays(value);
```

Comment:
Create the `useBookingSystem` hook

Reply:
Can You please specify what should be included in said hook?